core/src/main/web/app/app.scss

```diff
@@ -342,6 +342,9 @@ background-color: rgb(205, 223, 214);
   height: auto;
 }
 
+.modal-footer.fixed > span:nth-of-type(2){
```

Comment:
why so fancy?
isn't it better to add a style to the element and then access it by name?